resources/views/items/index.blade.php

```diff
@@ -57,6 +81,7 @@
     $(document).ready(function() {
         $('.is_active_switch').bootstrapSwitch();
 
+        // Handle switch click using Ajax call
         $('.is_active_switch').on('switchChange.bootstrapSwitch', function(event, state) {
```

Comment:
kebab case for html/css classes
snake case for API keys of JSON
camel case for function and variable names